scripts/accessibility-check.js

```diff
@@ -1,221 +1,91 @@
+        // Dark mode has lots of errors. It is still very WIP so ignore for now
+        // `${url}&contexts=New%20Design%20Language%3DEnabled%20-%20Dark%20Mode`,
+      );
+    }
 
-    await initialPage.goto(iframePath);
-    const stories = await initialPage.evaluate(() => {
-      return window.__STORYBOOK_CLIENT_API__.raw();
-    });
+    return memo;
+  }, []);
 
-    await initialPage.close();
+  return urls;
+}
 
-    const storyUrls = stories.reduce((memo, story) => {
-      // There is no need to test the Playground, or the "All Examples" stories
-      const isSkippedStory =
-        story.kind === 'Playground/Playground' || story.name === 'All Examples';
+const formatMessage = (id, violations) => {
+  const url = chalk.underline.blue(`http://localhost:6006/iframe.html?id=all-components-${id}`);
+  return violations
+    .map(fail => ` - ${chalk.bgRed.bold(fail.impact)} ${fail.description}\n   ${url}`);
+}
 
-      if (!isSkippedStory) {
-        const idParam = `id=${encodeURIComponent(story.id)}`;
-        memo.push(
-          idParam,
-          `${idParam}&contexts=Global%20Theming=Enabled%20-%20Light%20Mode`,
-          // Dark mode has lots of errors. It is still very WIP so ignore for now
-          // `${idParam}&contexts=Global%20Theming=Enabled%20-%20Dark%20Mode`,
-        );
-      }
-      return memo;
-    }, []);
+(async () => {
+  try {
+    //  Open a browser
+    console.log(chalk.bold(`💻 Opening ${concurrentCount} pages in chromium`));
+    const browser = await puppeteer.launch();
 
-    /**
-     * @param {string} url
-     */
-    const testPage = async (url) => {
-      try {
-        console.log(`Testing: ${url}`);
-        const page = await browser.newPage();
-        await page.goto(`${iframePath}?${url}`);
+    // Get the test ids from storybook
+    const testIds = await getUrls(browser);
 
-        const result = await page.evaluate(() => {
-          return window.axe.run(document.getElementById('root'), {});
-        });
+    console.log(chalk.bold(`✨ Testing ${testIds.length} pages with axe`));
 
-        await page.close();
+    // Test the pages with axe
+    const testPage = async (url) => {
+      const id = url.replace(`${iframePath}?id=all-components-`, '');
+      console.log(` - ${id}`);
 
-        if (result.violations.length === 0) {
-          return Promise.resolve({type: 'PASS', url, errorCount: 0});
-        }
+      const page = await browser.newPage();
+      await page.goto(url);
+      const result = await page.evaluate(() => window.axe.run(document.getElementById('root'), {}));
+      await page.close();
 
-        return Promise.resolve({
-          type: 'FAIL',
-          url,
-          errorCount: result.violations.length,
-          error: JSON.stringify(result.violations, null, 2),
-        });
-      } catch (error) {
-        return Promise.resolve({
-          type: 'ERROR',
-          url,
-          error: JSON.stringify(error, null, 2),
-        });
```

Comment:
We don't need to resolve a promise inside `p-map`. Removing the different types as the user only needs to know if there is an error caught or if the a11y tests failed.